src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -642,7 +654,8 @@ void GenerateFieldAccessor(const FieldDescriptor* field, const Options& options,
         "$this->readWrapperValue(\"^field_name^\");\n"
         "}\n\n",
         "camel_name", UnderscoresToCamelCase(field->name(), true), "field_name",
-        field->name(), "deprecation_trigger", deprecation_trigger);
+        field->name(), "deprecation_trigger",
+        deprecation_trigger_with_conditional);
```

Comment:
the field getter here may throw an error for wrapped fields since we aren't calling `readWrapperValue`

Reply:
fixed in 3d3e299